modules/gdnative/nativescript/godot_nativescript.cpp

```diff
@@ -260,17 +237,10 @@ void GDAPI godot_nativescript_set_method_documentation(void *p_gdnative_handle,
 	String *s = (String *)p_gdnative_handle;
 
 	Map<StringName, NativeScriptDesc>::Element *E = NSL->library_classes[*s].find(p_name);
-
-	if (!E) {
-		ERR_EXPLAIN("Attempted to add documentation to a method on a non-existent class!");
-		ERR_FAIL();
-	}
+	ERR_FAIL_COND_MSG(!E, "Attempted to add documentation to a method on a non-existent class!");
 
 	Map<StringName, NativeScriptDesc::Method>::Element *method = E->get().methods.find(p_function_name);
-	if (!method) {
-		ERR_EXPLAIN("Attempted to add documentatino to non-existent method!");
-		ERR_FAIL();
-	}
+	ERR_FAIL_COND_MSG(!method, "Attempted to add documentatino to non-existent method!");
```

Comment:
Typo in documentation.

The exclamation marks in this class are maybe a bit overkill too.

Reply:
There's a few other places where exclamation marks occur, should those be changed as well?

Reply:
I'd say yes, I don't see many cases where it would actually be a good practice.

Reply:
At least for that given file I went through and replaced the instances of exclamation marks in the error messages with simple periods then :)